From 12bd4eee1bb882417a38c2b5c9790e97725e489d Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 3 Sep 2020 15:12:50 +0100
Subject: [PATCH] dt: Remove duplicate assignment for i2c0 pinctrl
 config

The include file bcm283x-rpi-i2c0mux_0_XX.dtsi was setting
pinctrl-0 on i2c0mux, as were the individual platform DT files.

Remove this duplication and rely on the include file.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2708-rpi-b-plus.dts   | 4 ----
 arch/arm/boot/dts/bcm2708-rpi-b.dts        | 4 ----
 arch/arm/boot/dts/bcm2708-rpi-cm.dts       | 4 ----
 arch/arm/boot/dts/bcm2708-rpi-zero-w.dts   | 4 ----
 arch/arm/boot/dts/bcm2708-rpi-zero.dts     | 4 ----
 arch/arm/boot/dts/bcm2709-rpi-2-b.dts      | 4 ----
 arch/arm/boot/dts/bcm2710-rpi-2-b.dts      | 4 ----
 arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts | 4 ----
 arch/arm/boot/dts/bcm2710-rpi-3-b.dts      | 4 ----
 arch/arm/boot/dts/bcm2710-rpi-cm3.dts      | 4 ----
 10 files changed, 40 deletions(-)

--- a/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
@@ -73,10 +73,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2708-rpi-b.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b.dts
@@ -73,10 +73,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2708-rpi-cm.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-cm.dts
@@ -72,10 +72,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
@@ -122,10 +122,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2708-rpi-zero.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero.dts
@@ -76,10 +76,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
+++ b/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
@@ -73,10 +73,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2710-rpi-2-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-2-b.dts
@@ -73,10 +73,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
@@ -132,10 +132,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
@@ -147,10 +147,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
--- a/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
@@ -92,10 +92,6 @@
 	clock-frequency = <100000>;
 };
 
-&i2c0mux {
-	pinctrl-0 = <&i2c0_pins>;
-};
-
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
